### PR TITLE
Add to cal and iframe sizer

### PR DESCRIPTION
**Pull request by subaha-cob**
Add to cal and modifications to iframe auto sizer

### CODE REVIEW

File: docroot/themes/custom/bos_theme/dist/js/atc.min.js [hunk 1]
```diff
@@ -0,0 +1 @@
...t.readyState||!document.attachEvent&&"interactive"===document.readyState?setTimeout(d,1):o||(document.addEventListener?(document.addEventListener("DOMContentLoaded",d,!1),window.addEventListener("load",d,!1)):(document.attachEvent("onreadystatechange",c),window.attachEvent("onload",d)),o=!0))}}("addeventReady",window);var ate_touch_capable="ontouchstart"in window||window.DocumentTouch&&document instanceof window.DocumentTouch||0<navigator.maxTouchPoints||0<window.navigator.msMaxTouchPoints;window.addEventListener?(document.addEventListener("click",addeventatc.documentclick,!1),ate_touch_capable&&document.addEventListener("touchend",addeventatc.documentclick,!1)):window.attachEvent?(document.attachEvent("onclick",addeventatc.documentclick),ate_touch_capable&&document.attachEvent("ontouchend",addeventatc.documentclick)):document.onclick=function(){addeventatc.documentclick(event)},addeventReady(function(){addeventatc.initialize()});var flbckcnt=0,flbckint=setInterval(function(){15<=++flbckcnt||addeventatc.isloaded()?clearInterval(flbckint):addeventatc.initialize()},300),nlbckcnt=0,nlbckint=setInterval(function(){15<=++nlbckcnt?clearInterval(nlbckint):addeventatc.notloadedcnt()},300);
```
Comment (davidrkupton) [inline docroot/themes/custom/bos_theme/dist/js/atc.min.js:1]: Is this file needed?

**Review by davidrkupton — APPROVED** [review]
LGTM

File: docroot/themes/custom/bos_theme/dist/js/atc.min.js (hunk 1 above)
Comment (subaha-cob) [reply docroot/themes/custom/bos_theme/dist/js/atc.min.js:1]: No it is not. That was a test i should have deleted. Deleting it in my next push.
